recipes/systemd_service.rb

```diff
@@ -42,6 +42,11 @@ class ::Chef::Recipe
 
 service_unit_content['Service'].delete('Restart') if timer
 
+if node['chef_client']['systemd']['timeout']
```

Comment:
@iennae

From the [official docs](https://www.freedesktop.org/software/systemd/man/systemd.service.html#TimeoutStartSec=):

> Defaults to DefaultTimeoutStartSec= from the manager configuration file, except when Type=oneshot is used, in which case the timeout is disabled by default

Maybe I get it wrong, from my understanding it means that oneshot services does not have timeout per default, but everything else does. So setting it this way should be ok?